docs/user/aws/install_upi.md

```diff
@@ -0,0 +1,276 @@
+and load balancer configuration.
+
+## Monitor for `bootstrap-complete` and Initialization
+
+```console
+$ bin/openshift-install user-provided-infrastructure bootstrap-complete
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
+INFO API v1.12.4+c53f462 up
+INFO Waiting up to 30m0s for the bootstrap-complete event...
+```
+
+## Destroy Bootstrap Resources
+
+At this point, you should delete the bootstrap resources. If using the CloudFormation template, you would [delete the
+stack][delete-stack] created for the bootstrap to clean up all the temporary resources.
+
+## Cleanup Machine API Resources
+
+By querying the Machine API, you'll notice the cluster is attempting to reconcile the predefined
+Machine and MachineSet definitions. We will begin to correct that here. In this step, we delete
+the pre-defined master nodes. Our masters are not controlled by the Machine API.
+
+### Example: Deleting Master Machine Definitions
+
+```console
+$ export KUBECONFIG=auth/kubeconfig
+$ oc get machines --namespace openshift-machine-api
+NAME                                    INSTANCE   STATE     TYPE        REGION      ZONE         AGE
+test-tkh7l-master-0                                       m4.xlarge   us-east-2   us-east-2a   9m22s
+test-tkh7l-master-1                                       m4.xlarge   us-east-2   us-east-2b   9m22s
+test-tkh7l-master-2                                       m4.xlarge   us-east-2   us-east-2c   9m21s
+test-tkh7l-worker-us-east-2a-qjcxq                        m4.large    us-east-2   us-east-2a   8m6s
+test-tkh7l-worker-us-east-2b-nq8zs                        m4.large    us-east-2   us-east-2b   8m6s
+test-tkh7l-worker-us-east-2c-ww6c6                        m4.large    us-east-2   us-east-2c   8m7s
+$ oc delete machine --namespace openshift-machine-api test-tkh7l-master-0
+machine.machine.openshift.io "test-tkh7l-master-0" deleted
+$ oc delete machine --namespace openshift-machine-api test-tkh7l-master-1
+machine.machine.openshift.io "test-tkh7l-master-1" deleted
+$ oc delete machine --namespace openshift-machine-api test-tkh7l-master-2
+machine.machine.openshift.io "test-tkh7l-master-2" deleted
+```
+
+## Launch Additional Worker Nodes
+
+To launch workers, you are able to launch individual EC2 instances discretely or by automated processes outside the
+cluster (e.g. Auto Scaling Groups). However, you are also able to take advantage of the built in cluster scaling mechanisms
+and the machine API in OCP.
+
+### Option 1: Dynamic Compute using Machine API
+
+By default, MachineSets are created and will have failed to launch. We can correct the desired subnet filter,
+target security group, RHEL CoreOS AMI and EC2 instance profile.
+
+```console
+$ oc get machinesets --namespace openshift-machine-api
+NAME                              DESIRED   CURRENT   READY     AVAILABLE   AGE
```

Comment:
nit: adjust these headers so they line up with the data below (whose alignment changed with b29c9736a -> dafdaa034)?

Reply:
> nit: adjust these headers so they line up...

Done with dafdaa034 -> bdae4f939.